flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/StreamingFileSink.java

```diff
@@ -401,6 +402,7 @@ public T withOutputFileConfig(final OutputFileConfig outputFileConfig) {
 					bucketFactory,
 					new BulkPartWriter.Factory<>(writerFactory),
 					rollingPolicy,
+					null,
```

Comment:
Add setter to this bulk part writer builder?

Reply:
I think we may only first add the listener for internal user this time ? We may publish the listen also to users' API when we are more confirm with the listener interface in the future.

Reply:
For the table file system connector, it uses BulkWriter and Encoder to write files, so it need this listener too.

Reply:
Added the listener to the format builder to allow users to use it.